docs/modules/ROOT/pages/adr/0031-naming-scheme-for-servala-cluster-names-and-urls.adoc

```diff
@@ -0,0 +1,75 @@
+= ADR 0031 - Naming scheme for servala cluster names and URLs
+:adr_author:    Nicolas Bigler
+:adr_owner:     Schedar
+:adr_reviewers: 
+:adr_date:      2025-04-11
+:adr_upd_date:  2025-04-11
+:adr_status:    draft
+:adr_tags:      servala
+
+include::partial$adr-meta.adoc[]
+
+[NOTE]
+.Summary
+====
+With the launch of Servala we will have multiple clusters running on different cloud providers and on different regions. To ensure consistency and ease of management, it is important to establish a naming scheme for these clusters. This ADR outlines the proposed naming scheme for servala cluster names and URLs.
+====
+
+== Context
+
+=== Cluster names
+
+We have two different kind of clusters for servala:
+* Control-plane clusters: Those run crossplane and contain all the crossplane resources like functions, composites, api-server etc.
+* Service clusters: Those run the workloads that are managed by crossplane.
+
+To properly distinguish and identify each cluster, the cluster name should contain the following information:
+* The CSP where the cluster is running: Cloudscale, Exoscale, etc.
+* The region where the cluster is running: `ch-gva-2`, `lpg1`, etc.
+* The type of cluster: `control` or `service`
+* The stage of the cluster: `test`, `prod`
+* A counter for each cluster to ensure uniqueness.
+
+Lieutenant already gives us part of the naming scheme. Each cluster should be prefixed with `c-` followed by the tenant, so that it can easily identify which tenant owns the cluster.
+
+=== URLs
+
+We have to differnt URLs that are important:
+* The clusters API endpoint
+* The apps endpoint of Openshift for applications on the service clusters
+
+The defaults for Openshift are:
+
+* `api.[clustername].domain.tld`
+* `*.apps.[clustername].domain.tld`
+
+== Decision
+
+=== Cluster names
+
+Putting all the information above together we propose the following naming scheme for servala cluster names:
+
+`c-servala-[type]-[csp]-[region]-[stage][counter]`
+
+Examples:
+
+* First production control-plane cluster running on cloudscale in the lgp1:
+`c-servala-control-cloudscale-lpg1-prod1`
+
+* Second test service cluster running on exoscale in the ch-gva-2 region:
+`c-servala-service-exoscale-ch-gva-2-test2`
+
+=== URLs
+
+We have a dedicated domain for servala clusters: servala.com and will use that domain for all clusters
```

Comment:
As this domain is also used for the website, portal, etc. I suggest moving to a subdomain, e.g.:

* Control-plane clusters: *.control.servala.com
* Service cluster *.services.servala.com

Reply:
I already suggested the subdomain for control.servala.com.
For the service clusters I wouldn't use services.servala.com as it will just unnecessarily increase the domain name and the default apps domain on openshift already is `*.apps.<cluster_name>.<domain>`

Reply:
Ok, I think this is good enough